reconcile/utils/saasherder.py

```diff
@@ -453,7 +488,7 @@ def _process_template(self, options):
                 logging.error(
                     f"[{url}/{path}:{target_ref}] " +
                     f"error fetching directory: {str(e)}")
-                return None, None
+                return None, None, None
```

Comment:
Why is this needed?

Reply:
Prevents these error conditions from raising `ValueError: not enough values to unpack (expected 3, got 2)` from the [caller](https://github.com/app-sre/qontract-reconcile/blob/035460ca61d130aab188db6359c602db1b2bf92d/reconcile/utils/saasherder.py#L682-L683).

Reply:
(That's the "latent buglet" referenced by the [first commit](https://github.com/app-sre/qontract-reconcile/pull/1486/commits/aa54ee96618804f0f2991e044efd973719154348))

Reply:
OH NICE